define/helper.go

```diff
@@ -0,0 +1,20 @@
+package define
+
```

Comment:
I don't think we currently use those functions, but kept them in case they become useful later on